src/test/bundler_test.ts

```diff
@@ -196,9 +196,9 @@ suite('Bundler', () => {
     const doc = await bundle('test/html/import-in-body.html');
     const imports = dom5.queryAll(doc, importMatcher);
     assert.equal(imports.length, 0);
-    const bodyContainer = dom5.query(doc, bodyContainerMatcher);
-    const scriptActual = dom5.query(doc, scriptExpected)!.parentNode;
-    const divActual = dom5.query(doc, divExpected)!.parentNode;
+    const bodyContainer = dom5.query(doc, bodyContainerMatcher)!;
```

Comment:
Yeah this just started.   Now anything cut from master is broken without the fix.